docs/openshift-ai/other-projects/configure-jupyter-notebook-use-gpus-aiml-modeling.md

```diff
@@ -0,0 +1,377 @@
+# Configure a Jupyter notebook to use GPUs for AI/ML modeling
+
+**Prerequisites**:
+
+Prepare your Jupyter notebook server for using a GPU, you need to have:
+
+- Select the correct data science project and create workbench, see
+[Populate the data science project](../data-science-project/using-projects-the-rhoai.md#populate-the-data-science-project)
+for more information.
+
+Please ensure that you start your Jupyter notebook server with options as depicted
+in the following configuration screen. This screen provides you with the opportunity
+to select a notebook image and configure its options, including the number of GPUs.
+
+![PyTorch Workbech Information](images/pytorch-workbench.png)
```

Comment:
if you cut off the white space on the right the rest of the text will be bigger and easier to read.